test/views/pages/map/static_map_test.dart

```diff
@@ -41,4 +44,42 @@ void main() {
       );
     });
   });
+
+  group("hash and == works", () {
+    test("hash", () {
+      const mapInfo1 = MapInfo(
+        currentLocation: LatLng(0, 0),
+        selectOption: MapSelectionOptions.nearby,
+      );
+      const mapInfo2 = MapInfo(
+        currentLocation: LatLng(1, 1),
+        selectOption: MapSelectionOptions.nearby,
+      );
+      const mapInfo3 = MapInfo(
+        currentLocation: LatLng(0, 0),
+        selectOption: MapSelectionOptions.nearby,
+      );
+
+      expect(mapInfo1.hashCode, isNot(mapInfo2.hashCode));
+      expect(mapInfo1.hashCode, mapInfo3.hashCode);
+    });
```

Comment:
I am not sure that this is exactly how we want to test `hashcode`.
Because it is not forbidden for two different classes to have the same hash (even if it is very unlikely).
I believe we could follow an implementation similar to the tests for `PostOverview` where we check that the hashcode function computes the hash of its parameters. (those are located under `test/models/ui/post_overview_test.dart`)
Also, I believe those tests (`hashcode` and `==`) should be placed under
`test/models/ui/map_info_test.dart`.

Reply:
Fixed with deda72b!